src/python_bindings/opt_to_py.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <typeindex>
+
+#include <boost/any.hpp>
+#include <pybind11/pybind11.h>
+
+namespace python_bindings {
+pybind11::object opt_to_py(std::type_index type, boost::any val);
```

Comment:
```suggestion
pybind11::object OptToPy(std::type_index type, boost::any val);
```
see [here](https://google.github.io/styleguide/cppguide.html#Function_Names)